pipeline/talend_test/test_configs/anmn_nsw.yaml

```diff
@@ -0,0 +1,60 @@
+
+
+name: anmn_nsw
+type: pipeline_version_2
+po: marty
+actions:
+  - type: ADD
+    files:
+    - dest: /mnt/ebs/incoming/ANMN/NSW
+      local_file: PH100.zip
+  - type: UPDATE
+    files:
+    - dest: /mnt/ebs/incoming/ANMN/NSW
+      local_file: PH100.updated.zip
+      remote_file: PH100.zip
+  - type: DELETE
+    path: IMOS/ANMN/NSW/PH100
```

Comment:
In this case I think it's enough to delete a couple of the files, but you need the full path. I would suggest deleting these:
```
IMOS/ANMN/NSW/PH100/Velocity/IMOS_ANMN-NSW_AETVZ_20170227T190000Z_PH100_FV01_PH100-1702-Sentinel-or-Monitor-Workhorse-ADCP-109.5_END-20170527T030500Z_C-20180117T000000Z.nc
IMOS/ANMN/NSW/PH100/Biogeochem_timeseries/burst-averaged/IMOS_ANMN-NSW_BCDKOSTUZ_20170227T015944Z_PH100_FV02_PH100-1702-WQM-15-burst-averaged_END-20170415T001455Z_C-20170905T161314Z.nc
IMOS/ANMN/NSW/PH100/Biogeochem_profiles/IMOS_ANMN-NSW_CDEKOSTZ_20170716T224142Z_PH100_FV01_Profile-SBE19plus_C-20171103T040301Z.nc
IMOS/ANMN/NSW/PH100/Temperature/IMOS_ANMN-NSW_TZ_20170224T130000Z_PH100_FV01_PH100-1702-Aqualogger-520PT-104_END-20170527T065500Z_C-20170905T035920Z.nc
```
Note that in the case of the anmn_adcp_timeseries harvest, this is deleting the file that was updated, so we would only be testing all the functionality if we compared the content of the db after each step, not just at the end.

Reply:
thanks @mhidas . Actually this raises something... one of the advantages that I saw of adding the `path` option is that it can function as a clean-up as well as DELETE action. 

I think a clean-up would be good so that we don't end up littering buckets with objects. 
Any thoughts @jonescc ?

Perhaps in the case of this test config the solution would be:
```
- type: DELETE
  file: IMOS/ANMN/NSW/PH100/Velocity/IMOS_ANMN-NSW_AETVZ_20170227T190000Z_PH100_FV01_PH100-1702-Sentinel-or-Monitor-Workhorse-ADCP-109.5_END-20170527T030500Z_C-20180117T000000Z.nc
- type: DELETE
  file: IMOS/ANMN/NSW/PH100/Biogeochem_timeseries/burst-averaged/IMOS_ANMN-NSW_BCDKOSTUZ_20170227T015944Z_PH100_FV02_PH100-1702-WQM-15-burst-averaged_END-20170415T001455Z_C-20170905T161314Z.nc
- type: DELETE
  file: IMOS/ANMN/NSW/PH100/Biogeochem_profiles/IMOS_ANMN-NSW_CDEKOSTZ_20170716T224142Z_PH100_FV01_Profile-SBE19plus_C-20171103T040301Z.nc
- type: DELETE
  file: IMOS/ANMN/NSW/PH100/Temperature/IMOS_ANMN-NSW_TZ_20170224T130000Z_PH100_FV01_PH100-1702-Aqualogger-520PT-104_END-20170527T065500Z_C-20170905T035920Z.nc

- type: DELETE
  path: IMOS/ANMN/NSW/PH100
```